manager/container.go

```diff
@@ -187,15 +193,21 @@ func (c *containerData) GetProcessList() ([]v2.ProcessInfo, error) {
 		if err != nil {
 			return nil, fmt.Errorf("invalid virtual size %q: %v", fields[7], err)
 		}
+		cgroup, err := c.getCgroupPath(fields[11])
+		if err != nil {
+			return nil, fmt.Errorf("could not parse cgroup path from %q: %v", fields[10], err)
+		}
+		// Remove the ps command we just ran from cadvisor container.
+		// Not necessary, but makes the cadvisor page looks cleaner.
+		if !inRootContainer && c.info.Name == cgroup && fields[10] == "ps" {
```

Comment:
the ps was run in the cAdvisor container, not the container we're examining right now no?